eclass/cmake-utils.eclass

```diff
@@ -523,11 +523,18 @@ enable_cmake-utils_src_configure() {
 		SET (CMAKE_Fortran_COMPILE_OBJECT "<CMAKE_Fortran_COMPILER> <DEFINES> ${includes} ${FCFLAGS} <FLAGS> -o <OBJECT> -c <SOURCE>" CACHE STRING "Fortran compile command" FORCE)
 	_EOF_
 
+	local myCC=$(tc-getCC) myCXX=$(tc-getCXX) myFC=$(tc-getFC)
+
+	# !!! IMPORTANT NOTE !!!
+	# Single slash below is intentional. CMake is weird and wants the
+	# CMAKE_*_VARIABLES split into two elements: the first one with
+	# compiler path, and the second all with all command-line options,
```

Comment:
applied
$> patch -p1 < /root/1dbb40df4d570c8fed6ff564b312b7f1ec07cae6.patch
here onto /usr/portage of the tinderbox